pvlib/tests/test_atmosphere.py

```diff
@@ -82,12 +82,50 @@ def test_gueymard94_pw():
     pws = atmosphere.gueymard94_pw(temps_humids[:, 0], temps_humids[:, 1])
 
     expected = np.array(
-        [  0.1       ,   0.33702061,   1.12340202,   0.1       ,
+        [0.1       ,   0.33702061,   1.12340202,   0.1       ,
```

Comment:
I suggest reverting the changes to lines 85 and 36, just to prevent the linter from complaining at you

Reply:
hey @kandersolar thanks for the look and sorry for the late response, I have been out for the holiday.   I was wondering, should I try to remove all of the flake8 linter complaints?  I could revert how the function looks in main, but flake8 still doesn't like the white space, or I could remove the white space and make the linter happy..

Reply:
Please address the linter complaints for the new lines you are adding.  When the linter complains about existing code, it's often appropriate to just ignore it (and live with the little red X next to the linter job's status).  Sometimes those cases are false positives (i.e., we think the formatting is good as is, and would be worse if we changed it to satisfy the linter), but even if not, it's better to keep PRs focused on one thing at a time, and unrelated linter issues can always be addressed in follow-on PRs.

Reply:
reverted!